#gbu_release_node
g: 9.8                      
air_drag_coe:  0.3  
mass: 0.78
area: 0.0036
air_density: 1.225
dt: 0.01
D: 10
target: [24.7279065,121.7482841,30] # Long, Lat, Alt(Releated Alt)
B: 1.2
C: 10
